conu/apidefs/container.py

```diff
@@ -239,9 +239,10 @@ def stop(self):
 
     def kill(self, signal=None):
         """
-        kill this container
+        kill this container (bear in mind that the process won't have time to shutdown properly
```

Comment:
Just wondering. Killing isn't the only ability of this method. docker kill --help says `Kill one or more running containers` but I think I like `Send a signal to..` better. @TomasTomecek what do you think?

Reply:
I agree with you, will update.